ui/Modal/Funding/LinkAddress.svelte

```diff
@@ -1,11 +1,13 @@
 <script lang="typescript">
+  import * as svelteStore from "svelte/store";
+
   import EthToRadicle from "./Link/EthToRadicle.svelte";
   import EnterPassphrase from "./Link/EnterPassphrase.svelte";
   import SavedToRadicle from "./Link/SavedToRadicle.svelte";
   import RadicleToEth from "./Link/RadicleToEth.svelte";
   import { Remote } from "../../DesignSystem/Component";
 
-  import { wallet } from "../../src/wallet";
+  import { store as walletStore } from "../../src/wallet";
```

Comment:
Just a suggestion or something to consider for the future. If you use qualified imports like `import * as wallet` then you don’t need to alias exports and the import statement does not need to change if you are using more stuff.

Reply:
Completly aware of that. The reason why I go this way for stores is that if you qualify the import, you can't do `$wallet.store` but you can do `$walletStore`.